crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -128,6 +128,8 @@ impl Display for RetrievalLimit {
 }
 
 static VECTOR_DISTANCE_COLUMN_NAME: &str = "dist";
+static VSS_TEMP_GEN_ID_COLUMN: &str = "vss_temp_gen_id";
+static VSS_TEMP_TABLE_NAME: &str = "vss_temp_table";
```

Comment:
nit add comments explaining what these constants represent

Reply:
Added - thank you!